### PR TITLE
Add non-optimized builds

### PR DESCRIPTION
**Pull request by snormore**
And move optimized builds to `-optimized`-suffixed locations.

Addresses https://github.com/triagemd/tensorflow-docker/issues/2

### CODE REVIEW

File: python2.7-tensorflow-serving-gpu/Dockerfile [hunk 1]
```diff
@@ -33,12 +33,6 @@ RUN cd /opt && \
     cd .. && \
     pip install numpy grpcio && \
     bazel build -c opt \
```
Comment (stephensolis) [inline python2.7-tensorflow-serving-gpu/Dockerfile:35]: Why don't we just pull from PyPI in these cases?

Comment (snormore) [reply python2.7-tensorflow-serving-gpu/Dockerfile]: Definitely should

Comment (snormore) [reply python2.7-tensorflow-serving-gpu/Dockerfile:35]: ✅

Comment (stephensolis) [reply python2.7-tensorflow-serving-gpu/Dockerfile:35]: Sorry if this is a WIP, but you forgot the Python 2 images

Comment (snormore) [reply python2.7-tensorflow-serving-gpu/Dockerfile:35]: These are the tf-serving images, there are no PyPI packages for those. We only build Docker images for tensorflow itself on Python 3.6 for now.

Comment (stephensolis) [reply python2.7-tensorflow-serving-gpu/Dockerfile:35]: Ah, got it!

**Review by stephensolis — APPROVED** [review]

**Pull request merged**